### ISSUE TITLE
High CPU usage with no players connected

### ISSUE THREAD

**Issue by DavidFuchs** [lloesche/valheim-server-docker#44]
With no players connected to the server, my Docker container is running at high CPU:

```
CONTAINER ID        NAME                CPU %               MEM USAGE / LIMIT     MEM %               NET I/O             BLOCK I/O           PIDS
f32ddb31ba13        valheim             107.36%             1.62GiB / 15.34GiB    10.56%              0B / 0B             4.18GB / 10.9MB     46
```

The CPU is an older one (`Intel(R) Core(TM) i7-3770K CPU @ 3.50GHz`), but the CPU usage is still too high for a server with no users on it.

I found [this thread](https://valheimbugs.featureupvote.com/suggestions/158688/dedicated-server-cpu-usage-high-on-linux) about high CPU usage, which only makes itself apparent on the Linux dedicated server.

Hopefully, the issue will be resolved in the dedicated server software itself, but I wanted to post the issue here as well, in case it can be solved with a configuration setting. One of the users on the linked thread (Marx) guesses that it may be related to a framerate configuration setting - could that setting be applied in the Docker image?

**Comment by lloesche** [comment]
Duplicate of https://github.com/lloesche/valheim-server-docker/discussions/10

**Comment by InB4DevOps** [comment]
Fixed in 0.143.6
See https://valheim.fandom.com/wiki/Change_Log/Patch_Notes

**Comment by DeadEnded** [comment]
YES!
My CPU usage dropped from ~15% to ~2% when no users are connected.
Fantastic update!
No users are connected so I expect it will go back up when people are connected, but very happy idle dropped so much!
